apps/web/src/index.css

```
@@ -1 +1,18 @@
+ * so the pattern looks organic. The min height of 25 % keeps the
+ * bar visually present on the trough. */
+@keyframes eq-bar {
+  0%, 100% {
+    height: 25%;
+  }
+  50% {
+    height: 100%;
+  }
+}
+
+@theme {
```

Stylelint (17.10.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)